client/flutter/lib/pages/onboarding/onboarding_page.dart

```diff
@@ -1,9 +1,16 @@
+import 'package:WHOFlutter/api/user_preferences.dart';
+import 'package:WHOFlutter/pages/home_page.dart';
 import 'package:WHOFlutter/pages/onboarding/legal_landing_page.dart';
 import 'package:WHOFlutter/pages/onboarding/location_sharing_page.dart';
 import 'package:WHOFlutter/pages/onboarding/notifications_page.dart';
+import 'package:firebase_analytics/firebase_analytics.dart';
 import 'package:flutter/material.dart';
 
 class OnboardingPage extends StatefulWidget {
+  const OnboardingPage({Key key, @required this.analytics}) : super(key: key);
```

Comment:
per the conversation above (https://github.com/WorldHealthOrganization/app/pull/794/files#r401876791) we would want this to be a positional parameter for now. You can mark a TODO to make this a named parameter in the future